Runtime/Utilities/UX/Pointers/BaseControllerPointer.cs

```diff
@@ -505,7 +528,42 @@ public virtual float PointerOrientation
         public virtual void OnPreRaycast() { }
 
         /// <inheritdoc />
-        public virtual void OnPostRaycast() { }
+        public virtual void OnPostRaycast() 
+        {
+            if (grabAction != MixedRealityInputAction.None)// && InputSourceParent..SourceType == InputSourceType.Controller) <- revisit if there are issues
```

Comment:
This comment will probably cause more confusion than it's good for for anyone who has not worked on this?

Reply:
Bad comment left in while I was still testing, it didn't cause a problem :D